exercises/concept/savings-account/savings_account_test.rb

```diff
@@ -102,4 +102,8 @@ def test_years_before_desired_balance_for_large_start_balance
   def test_years_before_desired_balance_for_large_difference_between_start_and_desired_balance
     assert_equal 85, SavingsAccount.years_before_desired_balance(2_345.67, 12_345.678_9)
   end
+
+  def test_years_before_desired_balance_for_zero_balance
+    assert_raises(RuntimeError) { SavingsAccount.years_before_desired_balance(0, -Float::INFINITY) }
```

Comment:
Why negative infinity and not positive infinity?  We could use `1` here as easily, and I think it would not look to be so "special".   We can communicate this better, perhaps by doing something like this:

```ruby
  def test_years_before_desired_balance_for_zero_balance
    arbitrary_goal = rand(10) + 1
    assert_raises(RuntimeError) do
      SavingsAccount.years_before_desired_balance(0, arbitrary_goal)
    end
  end
```

Reply:
This arbitrary goal is made so that it is not possible to be 0, because a balance of 0 with a goal of 0 is valid and achievable.